### DISCUSSION TITLE
关于北邮石川老师团队的连接

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#360] [1️⃣ 新鲜事]
近期通过 [OpenHGNN ](https://mp.weixin.qq.com/s/ZTZrC72bRcxlpkfv1OoDXw)开源项目，再次连接到北邮的[石川老师团队](http://www.shichuan.org/ShiChuan_ch.html)，去年来过数据学院做客，并在 CCF 中国开源大会上当面交流过。

实验室旗下的 [OpenHGNN ](https://github.com/BUPT-GAMMA/OpenHGNN)开源项目，是一个异质图神经网络开源工具包，能够和开源数字生态数据，以及 OpenDigger 中的网络数据，有较好的协同。做该方向研究的可以参考与关注~ [OpenHGNN.pdf](https://github.com/X-lab2017/open-wonderland/files/14187509/OpenHGNN.pdf)

<div align=center>
<img src="https://github.com/X-lab2017/open-wonderland/assets/15010826/e9e3489b-d799-4751-a629-03b7a53ce4a0" width="650px">
</div>

**Comment by Nliver** [comment]
很酷的研究~
